core/src/main/scala/com/databricks/labs/remorph/generators/sql/ExpressionGenerator.scala

```diff
@@ -1,20 +1,29 @@
 package com.databricks.labs.remorph.generators.sql
 
-import com.databricks.labs.remorph.generators.GeneratorContext
-import com.databricks.labs.remorph.parsers.intermediate.Literal
+import com.databricks.labs.remorph.generators.{Generator, GeneratorContext}
+import com.databricks.labs.remorph.parsers.intermediate.{Expression, Literal}
```

Comment:
this import is redundant because of the following line - `intermediate => ir`

Reply:
good catch, thanks!